UPDATE.md

```diff
@@ -0,0 +1,66 @@
+To update your forked repository with the latest changes from the original repository (upstream), you can follow these steps using the command line. Make sure you have Git installed on your machine. Here's a step-by-step guide:
+
+### Step 1: Create a Backup Branch
+
+```bash
+# Assuming you are currently on your main branch
+git branch release_<current_version>
+
+# This will create a new branch named 'release_<current_version>' pointing to the same version tag as your current branch
+```
+
+### Step 2: Fetch the Latest tags from Upstream
+
+```bash
+# Fetch the latest tags from upstream
+git fetch upstream --tags
```

Comment:
Can we add also the step of making upstream the remote pointing to the upstream repository? In case someone doesn't did it before.

Reply:
Done 👍🏻